tests/test_rpm_verifier.py

```diff
@@ -319,9 +340,9 @@ def _mock_generate_output(with_failures: bool = False) -> MagicMock:
         return _mock_generate_output
 
     @pytest.fixture()
-    def status_path(self, tmp_path: Path) -> Path:
-        """Status temp file"""
-        return tmp_path / "status"
+    def temp_folder(self) -> Path:
+        """Create a temp folder"""
+        return Path(tempfile.mkdtemp())
```

Comment:
`tmp_path` is a ready-made fixture already providing what you're trying to do with `temp_folder`.
You should be able to just replace all your uses of `temp_folder` with `tmp_path` and delete the fixture.

Reply:
https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-fixture

Reply:
I reverted back to what it was, so for now I am not using the `tmp_path`
i will handle it in another PR